calculator/src/test/java/com/market/banica/calculator/service/JMXServiceImplTest.java

```diff
@@ -0,0 +1,125 @@
+package com.market.banica.calculator.service;
+
+import com.market.banica.calculator.data.contract.ProductBase;
+import com.market.banica.calculator.enums.UnitOfMeasure;
+import com.market.banica.calculator.model.Product;
+import com.market.banica.calculator.service.contract.ProductService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class JMXServiceImplTest {
+    public static final String PRODUCT_NAME = "banica";
+    public static final String INGREDIENT_NAME = "eggs";
+    public static final Integer QUANTITY = 3;
+    @Mock
+    private ProductService productService;
+    @Mock
+    private ProductBase productBase;
+
+    @InjectMocks
+    private JMXServiceImpl jmxService;
+
+
+    @Test
+    void getDatabaseShouldReturnHasMapWithKeyProductNameAndValueProduct() {
+        //Arrange
+        HashMap<String, Product> productsDatabse = new HashMap<>();
+        when(productBase.getDatabase()).thenReturn(productsDatabse);
```

Comment:
if you are not going to put anything in the map, you could define it in thenReturn(new HashMap<>())